vpr/src/place/place.cpp

```diff
@@ -1426,6 +1456,7 @@ static e_move_result try_swap(float t,
                                "ABORTED", "illegal move");
 
         move_outcome = ABORTED;
```

Comment:
Should be able to merge this code too:  just have one move_outcome that is used for both manual and regular moves.

Reply:
resolved